diff --git a/Mathlib/Analysis/LocallyConvex/Barrelled.lean b/Mathlib/Analysis/LocallyConvex/Barrelled.lean
--- a/Mathlib/Analysis/LocallyConvex/Barrelled.lean
+++ b/Mathlib/Analysis/LocallyConvex/Barrelled.lean
@@ -7,7 +7,6 @@

 public import Mathlib.Analysis.LocallyConvex.WithSeminorms
 public import Mathlib.Topology.Semicontinuity.Basic
-public import Mathlib.Topology.Baire.Lemmas

 /-!
 # Barrelled spaces and the Banach-Steinhaus theorem / Uniform Boundedness Principle
