diff --git a/Mathlib/Analysis/BoxIntegral/Box/Basic.lean b/Mathlib/Analysis/BoxIntegral/Box/Basic.lean
--- a/Mathlib/Analysis/BoxIntegral/Box/Basic.lean
+++ b/Mathlib/Analysis/BoxIntegral/Box/Basic.lean
@@ -5,13 +5,13 @@
 -/
 module

-public import Mathlib.Data.NNReal.Basic
-public import Mathlib.Order.Fin.Tuple
-public import Mathlib.Order.Interval.Set.Monotone
-public import Mathlib.Topology.MetricSpace.Basic
+import Mathlib.Data.NNReal.Basic
+import Mathlib.Order.Fin.Tuple
+import Mathlib.Order.Interval.Set.Monotone
+import Mathlib.Topology.MetricSpace.Basic
 public import Mathlib.Topology.MetricSpace.Bounded
-public import Mathlib.Topology.MetricSpace.Pseudo.Real
-public import Mathlib.Topology.Order.MonotoneConvergence
+import Mathlib.Topology.MetricSpace.Pseudo.Real
+import Mathlib.Topology.Order.MonotoneConvergence
 /-!
 # Rectangular boxes in `ℝⁿ`
