diff --git a/Mathlib/Analysis/LocallyConvex/PointwiseConvergence.lean b/Mathlib/Analysis/LocallyConvex/PointwiseConvergence.lean
--- a/Mathlib/Analysis/LocallyConvex/PointwiseConvergence.lean
+++ b/Mathlib/Analysis/LocallyConvex/PointwiseConvergence.lean
@@ -7,7 +7,7 @@

 public import Mathlib.Topology.Algebra.Module.Spaces.PointwiseConvergenceCLM
 public import Mathlib.Analysis.LocallyConvex.WithSeminorms
-public import Mathlib.Analysis.LocallyConvex.StrongTopology
+import Mathlib.Analysis.LocallyConvex.StrongTopology

 /-!
 # The topology of pointwise convergence is locally convex
